docs/components/modeler/desktop-modeler/flags/flags.md

```diff
@@ -109,7 +109,7 @@ As a result, the app will only allow users to model BPMN diagrams.
 
 <span class="badge badge--platform">Camunda 7 only</span>
 
-To disable the [history time to live hint](../../reference/modeling-guidance/rules/history-time-to-live.md) in scenarios where the engine configures HTTL, configure `flags.json`:
+To disable the history time to live hint in scenarios where the engine configures HTTL, configure `flags.json`:
```

Comment:
I think it's fine to leave the link here. We can adjust it `docs.camunda.org` but I think we should not get rid of it altogether.

Reply:
```suggestion
To disable the [history time to live](https://docs.camunda.org/manual/latest/modeler/history-time-to-live/) hint in scenarios where the engine configures HTTL, configure `flags.json`:
```